spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniparc/UniParcCrossReferenceDataStoreIndexer.java

```diff
@@ -1,21 +1,25 @@
 package org.uniprot.store.spark.indexer.uniparc;
```

Comment:
I couldn't see assert statement in UniParcCrossReferenceDataStoreIndexerTest for source.
Please note that i have not gone through all the tests. I will go during the second iteration.